index.html

```diff
@@ -125,15 +125,109 @@ <h2>Bez wychodzenia z domu</h2>
                    title="Wprowadź poprawny adres e-mail"><br>
             <input type="submit" value="Wyślij">
             <input type="reset" value="Wyczyść">
+        </form>
```

Comment:
po co to zamknięcie tagu form?

Reply:
webstorm pokazywał to jako błąd

Reply:
a czy na pewno powinno być tu? :)

Reply:
ok, mój błąd - github mi nie rozwinął całego kodu :)